src/factories/OrchestratorFactory_v1.sol

```diff
@@ -138,7 +139,14 @@ contract OrchestratorFactory_v1 is
         address proxy;
         //If the workflow should fetch their updates themselves
         if (workflowConfig.independentUpdates) {
-            //Use a InverterTransparentUpgradeableProxy as a proxy
+            //Deploy a proxy admin contract that owns the invidual proxies
+            //Overwriting the independentUpdateAdmin as the ProxyAdmin will
+            //be the actual admin of the proxy
+            workflowConfig.independentUpdateAdmin = address(
```

Comment:
Overwriting this in the config, as it is used down the line by the `createModule` function, thus using the `ProxyAdmin` as the admin, instead of the one that the user has set. The one the user set is used here, so the `ProxyAdmin` is owned by them.